codegen/gateway.go

```diff
@@ -315,54 +355,28 @@ func newClientSpec(
 }
 
 // MiddlewareSpec holds information about each middleware at the endpoint
-// level. The same mid
 type MiddlewareSpec struct {
 	// The middleware package name.
-	Name string `json:"name"`
-	// Go import path for the middleware.
-	Path string `json:"path,omitempty"`
+	Name string
 	// Middleware specific configuration options.
-	Options map[string]interface{} `json:"options,omitempty"`
+	Options map[string]interface{}
 	// Options pretty printed for template initialization
-	PrettyOptions map[string]string `json:"prettyOptions,omitempty"`
+	PrettyOptions map[string]string
```

Comment:
why required now?

Reply:
I believe this object is no longer serialized with my changes in the repository package